src/PrestaShopBundle/Bridge/AdminController/ControllerConfiguration.php

```diff
@@ -193,7 +194,37 @@ class ControllerConfiguration
     public $modals = [];
 
     /**
+     * This parameter is needed by legacy helper shop, we can't remove it.
+     *
      * @var int
      */
-    public $multishopContext = Shop::CONTEXT_ALL | Shop::CONTEXT_GROUP | Shop::CONTEXT_SHOP;
+    public $multishop_context = Shop::CONTEXT_ALL | Shop::CONTEXT_GROUP | Shop::CONTEXT_SHOP;
+
+    /**
+     * This parameter is needed by legacy hook, so we can't remove it.
```

Comment:
Perhaps it would be useful to state what it's used for

Reply:
Actually in Admin controllers it seems useless. I only found one usage at AdminLegacyLayoutController construct where it sets the value to controller name (altough there is also another property $controller_name holding same value).
So only usages might be in modules (probably some could rely on it to determine controller, but there are also other ways to do it e.g. checking instance). 
I personally would get rid of it here and in AdminLegacyLayoutController, but not sure about the BC. Maybe anyone else knows more? :thinking: